Kudu.Core/Helpers/PostDeploymentHelper.cs

```diff
@@ -163,6 +162,44 @@ public static async Task SyncFunctionsTriggers(string requestId, TraceListener t
 
             VerifyEnvironments();
 
+            // Try and let functions runtime call the settriggers
+            if (!await FunctionsRuntimeSyncTriggers(requestId))
+            {
+                Trace(TraceEventType.Information, "Attempting to perform settriggers call directly.");
+                await PerformSettriggers(requestId, functionsPath);
+            }
+
+            // this couples with sync function triggers
+            await SyncLogicAppJson(requestId, tracer);
+        }
+
+        private static async Task<bool> FunctionsRuntimeSyncTriggers(string requestId)
+        {
+            try
+            {
+                var scmUrl = IsLocalHost ? HttpAuthority : HttpHost;
```

Comment:
this should read scmHostName not scmUrl.

Reply:
Or perhaps "hostOrAuthority" as it is in the other location in this file where this code is present